20210703-CNS2021/02_git/Ankur.txt

```diff
@@ -0,0 +1 @@
+Ankur attended the session
```

Comment:
Could you update the text to use another name please?

Reply:
My bad @sanjayankur31 ! I made the changes. I hope it's correct now =)